test/e2e/run_userns_test.go

```diff
@@ -186,23 +180,22 @@ var _ = Describe("Podman UserNS support", func() {
 		session := podmanTest.Podman([]string{"run", "--userns=auto:size=500", "alpine", "cat", "/proc/self/uid_map"})
 		session.WaitWithDefaultTimeout()
 		Expect(session).Should(Exit(0))
-		ok, _ := session.GrepString("500")
```

Comment:
This, and (old) lines 194, 199 required manual fixing (my script could not identify them). Unless I'm missing something major, I believe these three lines were complete NOPs.